docs/Developer-Guide.md

```diff
@@ -85,6 +85,46 @@ You need to install the following to build Kata Containers components:
 - `make`.
 - `gcc` (required for building the shim and runtime).
 
+# Install a hypervisor
+
+When setting up Kata using a [packaged installation method](install/README.md#installing-on-a-linux-system), the `qemu-lite` hypervisor is installed automatically. For other installation methods, you will need to manually install a suitable hypervisor.
+
+## Build a custom QEMU
```

Comment:
Do any of these changes actually relate to s390? I did not find anything (but see earlier comment).